app/permissions/decidim/reporting_proposals/admin/permissions.rb

```diff
@@ -26,6 +29,10 @@ def component
           context[:proposal].try(:component) || context[:current_component]
         end
 
+        def user_author_note?
+          context[:proposal_note].author == user
```

Comment:
```suggestion
          context[:proposal_note].try(:author) == user
```
Is this not failing if the context note is missing?